### PR TITLE
Add types for the "iso-3166-1-alpha-2" npm package

### PR DESCRIPTION
**Pull request by Justus-D**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## iso-3166-1-alpha-2 ([<kbd>unpkg</kbd>](https://unpkg.com/browse/iso-3166-1-alpha-2@latest/))
was missing the following properties:
1. countries
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against fe891a08ab8726cc0661be4cb1c252e59786abe5
</p>

File: types/iso-3166-1-alpha-2/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for iso-3166-1-alpha-2 1.0
+// Project: https://github.com/georgzoeller/iso-3166-1-alpha-2
+// Definitions by: Justus Dietrich <https://github.com/Justus-D>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function getCountry(code: string): string | undefined;
+export function getCode(country: string): string | null;
+export function getCountries(): string[];
+export function getCodes(): string[];
+export function getData(): { [code: string]: string | undefined };
```
Comment (peterblazejewicz) [inline types/iso-3166-1-alpha-2/index.d.ts:10]: you can do this, as well, to export a symbol (as in original module), but that is optional:
```suggestion
declare class Iso31661a2 {
    getCountry(code: string): string | undefined;
    getCode(country: string): string | null;
    getCountries(): string[];
    getCodes(): string[];
    getData(): { [code: string]: string | undefined };
}

declare const singleton: Iso31661a2;
export = singleton;
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Justus-D thanks!

**Comment by Justus-D** [comment]
Ready to merge

**Pull request merged**